keps/sig-network/2004-topology-aware-subsetting/README.md

```diff
@@ -565,6 +578,162 @@ label. Each approach may group the endpoints in slightly different
 combinations of EndpointSlices, but it will still be just as easy to list all
 endpoints for a Service.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TopologyAwareSubsetting
+    - Components depending on the feature gate: kube-controller-manager
+  - [x] Other
+    - Describe the mechanism:
+      `endpointslice.kubernetes.io/subsetting` annotation on Service.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+      No
```

Comment:
For PRR reviewers - does this include "needs a restart to flip the gate" ?

Reply:
I think what we're saying here is:
- you can enable/disable feature by feature-gate switch (above - which requires restart)
- you can enable/disable the feature for your individual single Service (on service by service basis) - and that doesn't require restart

[Assuming that was the intention, can you please make it explicit that it's on Service by Service basis?]